packages/common/hangul/src/disassemble.ts

```diff
@@ -15,21 +15,21 @@ export function disassembleHangulToGroups(str: string) {
   for (const letter of str) {
     const disassembledComplete = disassembleCompleteHangulCharacter(letter);
 
-    if (disassembledComplete != null) {
```

Comment:
`slash` prefer shallow comparable then strict 
because of flexbility

Reply:
Thank you for the information!

And i fixed [it](https://github.com/toss/slash/pull/370/commits/9aaf0d8782f038b7a06c35ebda584a84cc527937).